app/components/app-header.hbs

```diff
@@ -1,3 +1,3 @@
 <header>
-  <h1>ember component patterns</h1>
-</header>
\ No newline at end of file
+  <h2><LinkTo @route='index'>{{t 'site_name_technical'}}</LinkTo></h2>
```

Comment:
Curious- why change the element?

Reply:
My thinking is we can only have one `<h1>` per page, and it may make sense to have the current page name be that than the same app header we use everywhere.

So that opens the question then, what should this app's `<h1>` be? Is it always the name of the site, or is it the current page title?

Reply:
Happy to open an issue if we feel this needs addressing, merging this so it's not held up :)